src/Main/Main.cpp

```diff
@@ -174,6 +174,9 @@ bool                 GRACKLE_CMB_FLOOR;
 bool                 GRACKLE_PE_HEATING;
 double               GRACKLE_PE_HEATING_RATE;
 char                 GRACKLE_CLOUDY_TABLE[MAX_STRING];
+int                  GRACKLE_THREE_BODY_RATE;
+int                  GRACKLE_CIE_COOLING;
```

Comment:
Declare `GRACKLE_THREE_BODY_RATE` and `GRACKLE_H2_OPA_APPROX` as `bool` instead of `int`, just to be consistent with other Grackle variables.